orc8r/cloud/go/obsidian/server/server.go

```
@@ -46,27 +45,17 @@
 	e.Use(CollectStats)
 	e.Use(middleware.Recover())
 
-	// With this, we implicitly override the static endpoint serving a static
-	// Swagger spec with a dynamic endpoint which dynamically serves the spec.
-	// Note that this behavior is not documented in the Echo framework.
-	if obsidian.EnableDynamicSwaggerSpecs {
-		yamlCommon, err := swagger.GetCommonSpec()
-		if err != nil {
-			glog.Errorf("Error retrieving Swagger common spec: %+v", err)
-		} else {
-			handler := handlers.GetGenerateCombinedSpecHandler(yamlCommon)
-			e.GET(obsidian.StaticURLPrefix+"/v1/swagger.yml", handler)
-		}
+	err := handlers.RegisterSwaggerHandlers(e)
+	if err != nil {
+		glog.Errorf("error occurred while registering Swagger handlers %+v", err)
```

- Add note on swallowing error
- "Error registering ..."